src/chains.jl

```diff
@@ -450,6 +538,11 @@ function header(c::Chains; section=missing)
         "= $(join(map(string, arr), ", "))\n"
     )
 
+    # Get the wall time
+    start = minimum(start_time(c))
+    stop = maximum(stop_time(c))
+    wall = wall_duration(c)
```

Comment:
With the suggestions above one can write:
```suggestion
    start = min_start(c)
    stop = max_stop(c)
    wall = wall_duration(c; start=start, stop=stop)
```